src/App.jsx

```diff
@@ -19,9 +19,22 @@ export function App() {
 	 * to create and join a new list.
 	 */
 	const [listToken, setListToken] = useStateWithStorage(
-		'my test list',
+		null,
 		'tcl-shopping-list-token',
 	);
+	/**
+	 * Callback function gets passed as a prop through Home component to retrieve generated token.
+	 * It is then set to localStorage through setListToken and added to the database.
+	 **/
+	const setList = async (token) => {
+		try {
+			setListToken(token);
+			const docToken = await addDoc(collection(db, `${token}`), {});
```

Comment:
how long does this `await`?
Should we put in a timeout and error out (to handle the case of firebase not responding?)

Reply:
Yes, I think this suggestion would fall under best practices.